src/ncp/spinel.h

```diff
@@ -2695,6 +2696,58 @@ typedef enum
 
     SPINEL_PROP_MESHCOP__END = 0x90,
 
+    SPINEL_PROP_SERVER__BEGIN = 0x90,
```

Comment:
Suggestion: How about using `0xA0-0xB0` for "SERVER"? Even though we don't currently use it currently the `0x90-0A0` and corresponding `0x1900` extension resgion were intended for OT specific properties.
https://github.com/openthread/openthread/blob/master/src/ncp/spinel.h#L793